appservice/src/intTest/groovy/nu/yona/server/EditGoalsTest.groovy

```diff
@@ -45,10 +45,6 @@ class EditGoalsTest extends AbstractAppServiceIntegrationTest
 		response.status == 200
 		response.responseData._links?.self.href == richard.url + appService.GOALS_PATH_FRAGMENT
 		response.responseData._embedded.budgetGoals.size() == 2
-		response.responseData._embedded.budgetGoals[1].activityCategoryName == 'gambling'
```

Comment:
Along with this correction: https://github.com/yonadev/yona-server/commit/d5fcac5c2631e9c5624a0ab2395ce87ebb873536 :(